src/Backend/Core/Installer/Data/locale.xml

```diff
@@ -7817,7 +7817,7 @@
       </item>
       <item type="label" name="PageTitle">
         <translation language="nl"><![CDATA[paginatitel]]></translation>
-        <translation language="en"><![CDATA[pagetitle]]></translation>
+        <translation language="en"><![CDATA[page title]]></translation>
```

Comment:
Shouldn't the Dutch translation also be split up?

Reply:
Nope, it is apparently one word in Dutch